.github/workflows/release.yml

```diff
@@ -30,3 +30,6 @@ jobs:
         uses: changesets/action@master
         env:
           GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+
+      - name: Run Prettier
+        run: yarn format
```

Comment:
I'd imagine we'd have to also commit these changes after, cuz right now it'd just make a dirty working tree.